exporter/googlemanagedprometheusexporter/config.go

```diff
@@ -54,6 +56,11 @@ func (c *GMPConfig) toCollectorConfig() collector.Config {
 	cfg.ProjectID = c.ProjectID
 	cfg.UserAgent = c.UserAgent
 	cfg.MetricConfig.ClientConfig = c.MetricConfig.ClientConfig
+	// add target_info and scope_info metrics
```

Comment:
Can we add config for enabling/disabling these, similar to the PRW exporer: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/config.go#L41 ?

Reply:
yup, added and enabled by default